AutoResxTranslator/MSTranslateService.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Net.Http;
+
+namespace AutoResxTranslator
+{
+    /// <summary>
+    /// Translation service using Microsoft Cogntive service.
+    /// 
+    /// ref: https://azure.microsoft.com/en-in/services/cognitive-services/translator-text-api/
+    /// </summary>
+    public class MSTranslateService
+    {
+        private static string host = "https://api-apc.cognitive.microsofttranslator.com";
+        private static string subscriptionKey = "<YOUR-SUBSCRIPTION-KEY-HERE>";
+        private static string region = "centralindia";
+
+        public static async System.Threading.Tasks.Task<string> TranslateAsync(
+            string text,
+            string fromLanguage,
+            string toLanguage)
+        {
+            if (fromLanguage.Equals("auto") || fromLanguage.Equals(""))
+            {
+                fromLanguage = null;
+            }
+
+            string route = "/translate?api-version=3.0&to=" + toLanguage + "&from=" + fromLanguage;
+
+            try
+            {
+                object[] body = new object[] { new { Text = text } };
+                var requestBody = JsonConvert.SerializeObject(body);
+
+                using (var client = new HttpClient())
```

Comment:
The project uses .net 4 you should have updated to v4.5 in the project to use HttpClient